internal/services/storage/storage_account_local_user_resource.go

```diff
@@ -0,0 +1,528 @@
+package storage
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2021-09-01/storage"
```

Comment:
is there a reason why we're using the Track1 SDK rather than `hashicorp/go-azure-sdk` here?